frontend/packages/component-library-styles/font.scss

```diff
@@ -133,6 +133,8 @@ $barlowSemiCondensed-semibold: var(--font-barlow-semi-condensed-semi-bold), 'Bar
 $barlowSemiCondensed-medium: var(--font-barlow-semi-condensed-medium), 'Barlow Semi Condensed Medium', $noto-sans-fonts,
   sans-serif;
 
+$code-font: 'Source Code Pro', Monaco, 'Bitstream Vera Sans Mono', 'Lucida Console', Terminal, monospace;
```

Comment:
This font is not a part of design system. We don't need it here.

It's back from 2015 and only used in one place. If you want to remove the original -font.scss file - you can try going to apps/style/common.scss line 1593 and replace `$function-name-font: $code-font;` with `$function-name-font: 'Source Code Pro', Monaco, 'Bitstream Vera Sans Mono', 'Lucida Console', Terminal, monospace;`. But we definetely don't want this font in design system

Reply:
I inlined the font and removed it